lib/src/package.dart

```diff
@@ -496,3 +507,21 @@ class PackageDep extends PackageName {
       samePackage(other) &&
       other.constraint == constraint;
 }
+
+/// The type of dependency from one package to another.
+class DependencyType {
```

Comment:
Nope. The toString is 'enumName.vauleName'. So if that won't work here – nevermind. 😄